src/shogun/mathematics/linalg/GPUBackend.h

```diff
@@ -66,6 +66,15 @@ class GPUBackend
 	 */
 	template <typename T>
 	T dot(const GPUVector<T> &a, const GPUVector<T> &b) const;
+
+	/**
+	 * Method that computes the sum of SGVectors using Eigen3
```

Comment:
not Eigen3.
I suggest to remove the library from the doc. Also above
@lambday thoughts?